### ISSUE TITLE
Better name for the "explore" screen

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-dc#25]
Yesterday we discussed the overall flow of 3 screens: Intro => More => Lab.  However, "More" is not a very descriptive term and I was wondering if we could come up with something better.  It seems the learning goal we wish to address is basically that objects in everyday life can be classified and conductors or insulators.  Is that correct?  Perhaps "Conductors and Insulators" is too verbose and obscure.  But how about "Objects"?

**Comment by ariel-phet** [comment]
Maybe "Explore"? so Intro, Explore, Lab. Or another word that suggest a bit more in depth than intro.

**Comment by samreid** [comment]
In my opinion, "more" and "explore" suffer from the same problem that they don't identify what there is "more" of or what you are "exploring".  Since you are exploring more objects, I thought "Objects" might be a good screen name, since the thing that the 2nd screen adds is primarily more objects.  Thoughts?

**Comment by ariel-phet** [comment]
@samreid if that is the issue "circuit elements" would be the correct nomenclature. Not a fan of "objects" - that is just kind of boring an vague. I don't quite agree that more and explore suffer from the same problem. "intro" doesn't tell you what you are "introing" to, however you are embedded within a circuit construction sim. So I think some progression like "intro, advanced, lab" or "intro, explore,lab" or such suggests a progression of depth of exploration.

**Comment by samreid** [comment]
How about "Conductors" as something less boring/vague that also gets at our learning goals and more specific about the content of that screen?

**Comment by arouinfar** [comment]
I definitely don't love "More" but thought of it as a placeholder name. I would agree with @ariel-phet that "Objects" is a bit bland. "Conductors and Insulators" is descriptive, but too verbose and "Conductors" is misleading. I think "Explore" is a viable option, though I think of Explore screens as being an intro, usually. 

Perhaps we could title the screen "Grab Bag" because it's really the focus. The Lab screen would also have the grab bag, but I think it's pretty typically for our Lab screens to include all features, so I don't think Intro - Grab Bag - Lab would be a misleading sequence.

**Comment by samreid** [comment]
After discussion, "Grab Bag" is my new favorite.  It also looks good in context (not too wordy) and will work great when it has a corresponding icon.

![image](https://cloud.githubusercontent.com/assets/679486/23571194/1e1056e0-0025-11e7-956e-691c5290d340.png)

**Comment by arouinfar** [comment]
I like the sequence of Explore - Grab Bag - Lab!

**Comment by ariel-phet** [comment]
Not to be a downer...but I do have one somewhat large concern with "grab bag" - I am not sure how well that idea translates into other languages, it seems like a very idiomatic expression in English. I would prefer if we could find something more universal.

**Comment by samreid** [comment]
I see your concern and think "grab bag" may be idiomatic and difficult to translate.  I'm not opposed to finding a better name for this, but I also skimmed the CCK Java translations for this to see how it had been translated before in case (a) it seems like there might not be a problem or (b) if there are good ideas there:

af: Item Sak
pt: Saco de Objectos
pt_br: Sacola Surpresa
es_pe: Caja de Sorpresas
French: panier à main (hand basket or maybe shopping cart)

**Comment by arouinfar** [comment]
@ariel-phet makes a good point. While @samreid has pointed out that "Grab Bag" appears to localize reasonably, I think the HTML5 version will differ a bit from Java. In Java, "Grab Bag" appears within the sim itself (next to a bag icon), so it's clear what the string refers to. For the HTML5 version, we've discussed adding the grab bag contents to the item carousel, so "Grab Bag" will only appear as the screen name. I think this will make i18n a bit more difficult.  

For now, I'm leaning towards Intro - Explore - Lab, but I will continue to think about possible alternatives for the the 2nd screen.

**Comment by samreid** [comment]
Brainstorming:
"Household Items"
or
"Everyday Items"

**Comment by samreid** [comment]
Naming this screen and picking the icon seem like they should be done together.  In https://github.com/phetsims/circuit-construction-kit-dc/issues/26 we are discussing a circuit with a coin.  I think "Explore" would be OK but I still feel that "Everyday Items" would be somewhat better.  I think a quick conversation during an upcoming design meeting would be enough to decide and move on.

**Comment by arouinfar** [comment]
Good idea @samreid. I've added this item to today's agenda.

**Comment by arouinfar** [comment]
As discussed 4/20/17, let's stick with Explore.
